packages/@uppy/companion/test/__tests__/uploader.js

```diff
@@ -190,9 +195,10 @@ describe('uploader with tus protocol', () => {
   })
 
   // eslint-disable-next-line max-len
-  const formDataNoMetaMatch = /^----------------------------\d+\r\nContent-Disposition: form-data; name="files\[\]"; filename="uppy-file-[^"]+"\r\nContent-Type: application\/octet-stream\r\n\r\nSome file content\r\n----------------------------\d+--\r\n$/
+  const formDataNoMetaMatch = /^--form-data-boundary-[a-z0-9]+\r\nContent-Disposition: form-data; name="files\[\]"; filename="uppy-file-[^"]+"\r\nContent-Type: application\/octet-stream\r\n\r\nSome file content\r\n--form-data-boundary-[a-z0-9]+--\r\n\r\n$/
 
   test('upload functions with xhr formdata', async () => {
+    delete process.env.UPPY_TEST_DO_NOT_WAIT_FOR_COMPLETE_BODY
```

Comment:
```suggestion
    // the hack breaks some tests, so we need to selectively disable it
    delete process.env.UPPY_TEST_DO_NOT_WAIT_FOR_COMPLETE_BODY
```

Reply:
Well it's not that it breaks the test, it's that the test validates the complete body, so we need to do wait for complete body.

Reply:
```suggestion
    // We want the complete body for this test:
    delete process.env.UPPY_TEST_DO_NOT_WAIT_FOR_COMPLETE_BODY
```